docs/setting-up-local-development.md

```diff
@@ -2,73 +2,187 @@
 
 This section walks you through getting the exercism.io app running locally.
 
-## Prerequisites
+- [Get the source code](#get-the-source-code)
+- [Install the prerequisites](#install-the-prerequisites)
+  - [Running exercism.io directly on your computer](#running-exercism-io-directly-on-your-computer)
+  - [Running exercism.io in a Docker container via Vagrant](#running-exercism-io-in-a-docker-container-via-vagrant)
+- [Initialize exercism.io](#initialize-exercism-io)
```

Comment:
The links `#initialize-exercism-io`, `#run-exercism-io-locally`, and `#configure` (which I presume should link to `Configure Your Local Copy of exercism.io` don't seem to be working. I'm not sure but markdown might be having trouble generating the links because of the period in exercism.io